src/analysis/statistical_analysis/flood_wave_analyzer.py

```diff
@@ -0,0 +1,57 @@
+import networkx as nx
+
+from src.analysis.statistical_analysis.stat_calculator import StatCalculator
+from src.graph_manipulation.flood_wave_filter import FloodWaveFilter
+
+
+class FloodWaveAnalyzer:
+    """
+    This class is responsible for the general statistical analysis of flood waves.
+    """
+    @staticmethod
+    def get_flood_wave_count_between_stations(extracted_graph: nx.DiGraph,
```

Comment:
Mivel kb. ugyanazokat az argumentumokat várja mindkét metódus, lehetne egy normális, nem statikus osztályunk, ahol a konstruktorban kérjük be ezeket és a metódusoknak már kb. nincs bemeneti argumentuma. (A `statistic` az kell az aggregáló metódushoz.)